governance.md

```diff
@@ -0,0 +1,137 @@
+- Have contributed a significant amount of work to the project (e.g. in the form of PRs or PR reviews), thereby demonstrating their trustworthiness and commitment to the project.
+
+New Peers can be nominated by any existing Peers. Once they have been nominated, there will be a vote by the Owners.
+
+It is important to recognize that being a Peer is a privilege, not a right. That privilege must be earned and once earned it can be removed by the Owners. However, under normal circumstances committership exists for as long as the Peer wishes to continue engaging with the project. Inactive Peers might be marked as inactive or removed by the Owners and may re-enter when they choose to contribute again.
+
+#### List of peers
+- Rachel Andrew (@rachelandrew)
+- Daniel Beck (@ddbeck)
+- Ryan Johnson (@escattone), Mozilla
+- Jean-Yves Perrier (@teoli2003), Mozilla
+- Eric Shepherd (@a2sheppy), Mozilla
+- Irene Smith (@irenesmith), Mozilla
+- Estelle Weyl (@estelle), Mozilla
+- John Whitlock (@jwhitlock), Mozilla
+
+Some Peers are also [MDN Content Curators](https://developer.mozilla.org/en-US/docs/MDN/Contribute/Documentation_topics_and_curators) and have thus earned the privilege to be a mdn-browser-compat-data peer, so that their expertise in a given content area (like CSS or HTML, etc.) can help improve the compat data for that same content area. Such peers are marked in the relevant folders using GitHub’s Code Owner mechanism.
+
+Peers might also be representatives of browser vendors and have expertise and/or access to browser-specific information within their company. Their company name is listed in the peer list.
+
+A Peer who shows an above-average level of contribution to the project, particularly with respect to its strategic direction and long-term health, may be nominated to become an Owner, described below.
+
+### Owners
+The mdn-browser-compat-data project is jointly governed by the [Mozilla MDN staff team](https://wiki.mozilla.org/Engagement/MDN_Durable_Team#Team_Members), the [MDN Product Advisory Board Members](https://developer.mozilla.org/en-US/docs/MDN/MDN_Product_Advisory_Board/Members), and the [Owner group](#list-of-owners). They are collectively responsible for high-level guidance of the project.
+
+The [Owner group](#list-of-owners) has final authority over this project including:
+
+- Technical direction of the project, especially infrastructure PRs and linting and/or schema decisions.
+- Project governance and process (including this policy).
+- Contribution policy.
+- GitHub repository hosting.
+
+Being an Owner is not time-limited. There is no fixed size of the Owner group. The Owner group should be of such a size as to ensure adequate coverage of important areas of expertise balanced with the ability to make decisions efficiently.
+An Owner may be removed from the Owner group by voluntary resignation, or by a standard Owner group motion.
```

Comment:
"standard Owner group motion" -> This could use a definition. Also, isn't remaining an Owner also subject to the same scrutiny as remaining a Peer, e.g. following the rules and not violating the code of conduct?